packages/application-shell/src/components/navbar/main-navbar.styles.ts

```diff
@@ -26,11 +26,29 @@ const FixedMenu = styled.div`
   }
 `;
 
+const SupportMenuTooltipContainer = styled.div`
+  position: fixed;
+  left: ${NAVBAR.sublistIndentationWhenCollapsed};
+  display: none;
+  align-items: center;
+  justify-content: flex-start;
+  height: ${NAVBAR.itemSize};
+  z-index: 20001;
+
+  ${MenuListItem} ${ItemIconText}
```

Comment:
I guess it's not used
```suggestion
```

Reply:
removed here: 8cfa7c8042987160af5a9a3eb15758732848f874